content/docs/hooks-state.md

```diff
@@ -187,6 +187,14 @@ In a function, we already have `setCount` and `count` as variables so we don't n
   </button>
 ```
 
+Alternatively, we could use another way of incrementing count variable by using `setCount` function parameter. This is in fact more safe way to update count variable, because we will make sure to receive the latest count variable from our state.
```

Comment:
https://reactjs.org/docs/hooks-reference.html#functional-updates we have it here?

Reply:
Yeah I think it’s reasonable to keep it in the API reference. In new docs it has its own page in the guides.